sklearn/ensemble/ensemble_selection.py

```diff
@@ -0,0 +1,261 @@
+"""
+Ensemble Selection Classifier
+
+This module contains a Ensemble Selection Classifier for
+fitted classification estimators.
+
+"""
+
+from collections import Counter
+
+import operator
+import numpy as np
+
+from ..base import BaseEstimator
+from ..base import ClassifierMixin
+from ..utils import check_random_state
+from ..metrics import accuracy_score, f1_score
+from sklearn.utils.validation import check_is_fitted
+
+
+def _accuracy(y, proba):
+    return accuracy_score(y, np.argmax(proba, axis=1))
+
+def _f1(y, proba):
+    return f1_score(y, np.argmax(proba, axis=1))
+
+
+class EnsembleSelectionClassifier(BaseEstimator, ClassifierMixin):
+    """An ensemble classifier built by greedy stepwise selection.
+
+    Parameters
+    ----------
+    estimators : list of (string, estimator) tuples
+        The estimators from which the ensemble selection classifier is built.
+        These estimators must be fitted.
```

Comment:
Please explain that the strings are arbitrary, unique names. I'm not sure they're well motivated, actually, if we're only dealing with fitted estimators: the model can refer to estimators by index.